src/Datadog.Trace/Tagging/TagsList.cs

```diff
@@ -276,6 +279,11 @@ private int WriteTags(ref byte[] bytes, int offset)
 
                     foreach (var pair in tags)
                     {
+                        if (pair.Key == Trace.Tags.Origin)
+                        {
+                            isOriginWritten = true;
```

Comment:
How is that possible since you check for that tag in the `SetTag` method?

Reply:
Oh you're right is not possible in `tags`, I'm removing it.

Reply:
done in d079b9e